compiler/optimizer/GlobalRegister_inlines.hpp

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2023 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef GLOBALREGISTER_INLINES_INCL
+#define GLOBALREGISTER_INLINES_INCL
+
+#include "optimizer/OMRGlobalRegister_inlines.hpp"
+
+#endif
```

Comment:
Thanks for the review! I didn't strongly consider moving all of TR_GlobalRegisterAllocator to OMR/TR namespace, but that was basically only because it wasn't strictly needed to allow the extension points I was trying to enable here. That being said, I'm not opposed to it, and I don't think that if other projects wanted it structured that way it would cause any problems in downstream projects that I'm aware of.

My understanding (for what it's worth; others with more experience with the OMR code than I may have additional thoughts) is that the tradeoff there is largely a matter of whether making a change to the structure of the class in that way may inadvertently lead to new bugs/disruptions to existing projects, against the consistency it would afford, both to OMR itself in having a consistent style and to downstream projects in having a more consistent set of extension points for the various optimization classes, to have all of the optimizations structured in a similar way (which at the moment actually isn't consistent across the various optimizations - some are TR_<opt>, others are in the OMR/TR namespace, others are fully extensible classes - but probably it would be nice if it were consistently one or another of those options).

Reply:
OK, that sounds reasonable.  Once you have a chance to add the missing newline, I'll mark my review as approved.

Reply:
I've added a newline to the end of the file.